linux/docker_virtualenv_ius_apache24.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e -u -x
+
+#OMEROVER=omero
+OMEROVER=omerodev
+
+source settings.env
+
+#install httpd 2.4
+yum -y install httpd24u
+
+#Create virtual env.
+virtualenv /tmp/omeroenv
```

Comment:
`/tmp` is a bad place since it implies it's temporary. How about `/home/omero/omeroenv`?